src/main/java/org/webshar/hrms/controllers/RoleController.java

```diff
@@ -0,0 +1,28 @@
+package org.webshar.hrms.controllers;
```

Comment:
Majorly singular package naming convention practice is there such as controller, service, repository instead of controllers, services, repositories, etc. We need to adhere to those conventions.